one-ui/src/main/java/com/marklogic/hub/oneui/auth/LoginLogoutHandler.java

```diff
@@ -27,14 +33,24 @@
 import java.io.IOException;
 
 public class LoginLogoutHandler implements AuthenticationSuccessHandler, LogoutSuccessHandler {
+    @Autowired
+    HubConfigSession hubConfigSession;
+
+    ObjectMapper mapper = new ObjectMapper();
 
     @Override
     public void onAuthenticationSuccess(HttpServletRequest request, HttpServletResponse response, Authentication authentication) throws IOException {
         ConnectionAuthenticationToken authenticationToken = (ConnectionAuthenticationToken)authentication;
-
+        ObjectNode output = mapper.createObjectNode();
         clearAuthenticationAttributes(request);
+        output.put("isInstalled", authenticationToken.stagingIsAccessible());
+        output.put("hasManagePrivileges", authenticationToken.hasManagePrivileges());
+        if (authenticationToken.stagingIsAccessible()) {
+            RolesService rolesService = RolesService.on(authenticationToken.getHubConfigSession().newStagingClient());
```

Comment:
https://github.com/marklogic/marklogic-data-hub/blob/f1305eb6fb34b989023422a53aca5523360ffaf7/one-ui/src/main/java/com/marklogic/hub/oneui/auth/LoginLogoutHandler.java#L49

Since we autowired the hubConfigSession, we do not need to access it through authentication token.

Reply:
I should remove the autowired hubConfig. I was getting a null hubConfig when I tried to use the autowired version.